HOW_TO_RELEASE.md

```diff
@@ -5,9 +5,9 @@
 01. Check that the branch you intend to release is passing tests on Travis.
     If it's not passing there you should not release it.
 
-00. Start from a completely clean conda environment 
-    and git repository.  Assuming you have `conda` installed, you can do so 
-    by starting where ActivitySim is not yet cloned (e.g. in an empty 
+00. Start from a completely clean conda environment
+    and git repository.  Assuming you have `conda` installed, you can do so
+    by starting where ActivitySim is not yet cloned (e.g. in an empty
```

Comment:
Why does this HOW_TO_RELEASE.md file have all these changes?  These changes look outside the scope of this PR and I am wondering if they were even included on purpose.  Can we revert this file back to the develop branch and handle these changes elsewhere?

Reply:
Thanks for catching this. It was not intentional. I have a couple of hypotheses on how it happened, but the correct file is now there.